ffs/api/api.go

```diff
@@ -209,6 +209,29 @@ func (i *API) Show(c cid.Cid) (ffs.CidInfo, error) {
 	return inf, nil
 }
 
+// ShowMatching returns a list of information about stored cids matching a selector
+func (i *API) ShowMatching(selector func(ffs.CidInfo) bool) ([]ffs.CidInfo, error) {
```

Comment:
Ok @jsign here is the implementation of that method to get all `CidInfo` matching a selector. 

There are a handful of ways the web ui work requires this data to be processed and returned to the client. Since those requirements are driven only by the web client at this point, I'm considering not implementing any further methods here on `ffs.API` and instead just using this method within the gRPC service and adding only new gRPC methods for the client. What do you think?

Reply:
Sounds good!